docs/book/v3/helpers/head-script.md

```diff
@@ -1,112 +1,37 @@
->     'text/javascript',
->     ['conditional' => 'lt IE 7']
-> );
-> ```
-
-> ### Preventing HTML style comments or CDATA wrapping of scripts
->
-> By default, `HeadScript` will wrap scripts with HTML comments or it wraps
-> scripts with XHTML CDATA.  This behavior can be problematic when you intend to
-> use the script tag in an alternative way by setting the type to something
-> other then `text/javascript`. To prevent such escaping, pass an `noescape`
-> with a value of true as part of the `$attrs` parameter in the method calls.
->
-> ```php
-> // jquery template
-> $template = '<div class="book">{{:title}}</div>';
->
-> $this->headScript()->appendScript(
->     $template,
->     'text/x-jquery-tmpl',
->     ['id' => 'tmpl-book', 'noescape' => true]
-> );
-> ```
-
-`HeadScript` also allows capturing scripts; this can be useful if you want to
-create the client-side script programmatically, and then place it elsewhere. The
-usage for this will be showed in an example below.
-
-Finally, you can also use the `headScript()` method to quickly add script
-elements; the signature for this is `headScript($mode = 'FILE', $spec = null,
-$placement = 'APPEND', array $attrs = [], $type = 'text/javascript')`. The
-`$mode` is either 'FILE' or 'SCRIPT', depending on if you're linking a script or
-defining one. `$spec` is either the script file to link or the script source
-itself. `$placement` should be either 'APPEND', 'PREPEND', or 'SET'. `$attrs` is
-an array of script attributes. `$type` is the script type attribute.
-
-`HeadScript` overrides each of `append()`, `offsetSet()`, `prepend()`, and
-`set()` to enforce usage of the special methods as listed above. Internally, it
-stores each item as a `stdClass` token, which it later serializes using the
-`itemToString()` method. This allows you to perform checks on the items in the
-stack, and optionally modify these items by modifying the object returned.
-
-The `HeadScript` helper is a concrete implementation of the
-[Placeholder helper](placeholder.md).
-
-> ### Use InlineScript for HTML Body Scripts
->
-> `HeadScript`'s sibling helper, [InlineScript](inline-script.md), should be
-> used when you wish to include scripts inline in the HTML **body**. Placing
-> scripts at the end of your document is a good practice for speeding up
-> delivery of your page, particularly when using 3rd party analytics scripts.
-
-> ### Arbitrary Attributes are Disabled by Default
->
-> By default, `HeadScript` only will render `<script>` attributes that are blessed by the W3C.
-> These include `id`, `charset`, `crossorigin`, `defer`, `integrity`,
-> `language`, `src`, and `type`. However, some JavaScript frameworks, notably
-> [Dojo](http://www.dojotoolkit.org/), utilize custom attributes in order to
-> modify behavior. To allow such attributes, you can enable them via the
-> `setAllowArbitraryAttributes()` method:
->
-> ```php
-> $this->headScript()->setAllowArbitraryAttributes(true);
-> ```
+# HeadScript & InlineScript
```

Comment:
Can we rename the file to something like `head-inline-script´ and add some redirects? The filename should nearly the same like the first headline.